### PR TITLE
Some of links updated! And added info

### PR DESCRIPTION
**Pull request by Hugovidafe**
Moving the end of a parenthesis to the right place for the link to work.
And added if a Github respository is archived, only-read.

### CODE REVIEW

**Comment by Hugovidafe** [comment]
Can we add comments after links with the date of when the link was last reviewed?

**Comment by Haroenv** [comment]
these changes look good, although archived projects should probably just be removed, could you do that?

**Comment by Hugovidafe** [comment]
Yeah, I can, but what abou React 360?

**Comment by Hugovidafe** [comment]
@Haroenvz, could we remove React 360? This is archived...

**Comment by Hugovidafe** [comment]
@Haroenv, could we remove React 360? Is archived...

File: README.md [hunk 1]
```diff
@@ -315,7 +315,7 @@ A collection of awesome things regarding the React ecosystem.
 - [compose-state](https://github.com/tvler/compose-state) - Compose multiple setState or getDerivedStateFromProps updaters in React
 - [react-lodash](https://github.com/typicode/react-lodash) - Lodash as React components
 - [react-helmet](https://github.com/nfl/react-helmet) - A document head manager for React
-- [Stator](https://github.com/cs01/stator) - Simple, plain JavaScript state management with built-in support for React
+- [Stator](https://github.com/cs01/stator) - Simple, plain JavaScript state management with built-in support for React (Archived, read-only)
```
Comment (Haroenv) [inline README.md:318]:
```suggestion
```

this one is small enough to remove

File: README.md [hunk 2]
```diff
@@ -498,7 +498,7 @@ A collection of awesome things regarding the React ecosystem.
 - [react-map-gl](https://github.com/uber/react-map-gl) - A React wrapper for MapboxGL-js plus overlay API
 - [google-map-react](https://github.com/istarkov/google-map-react) - Isomorphic google map React component
 - [react-mapbox-gl](https://github.com/alex3165/react-mapbox-gl) - A mapbox-gl-js wrapper to make the API react friendly
-- [google-maps-react](https://github.com/fullstackreact/ - A declarative Google Map React component using React, lazy-loading dependencies, current-location finder and a test-driven approach by the Fullstack React team.google-maps-react)
+- [google-maps-react](https://github.com/fullstackreact/) - A declarative Google Map React component using React, lazy-loading dependencies, current-location finder and a test-driven approach by the Fullstack React team.google-maps-react
```
Comment (Haroenv) [inline README.md:501]:
```suggestion
- [google-maps-react](https://github.com/fullstackreact/google-maps-react) - A declarative Google Map React component using React, lazy-loading dependencies, current-location finder and a test-driven approach by the Fullstack React team.
```

**Review by Haroenv — COMMENTED** [review]
makes sense, let's remove the really small archived one, and fix that typo though :)

**Comment by Hugovidafe** [comment]
Then, this gonna be good! We can merge it; after merge I'm gonna create a new issue asking for add comments for when the link was revised.

**Comment by Hugovidafe** [comment]
I think we need to add the new links here before merge, it can be?

**Comment by Haroenv** [comment]
Thanks a lot Hugo, this is good!

**Pull request merged**